grc/CMakeLists.txt

```diff
@@ -20,30 +20,17 @@ install(FILES
     dab_ofdm_sync_dab.xml
     dab_ofdm_sync_dab2.xml
     dab_detect_null.xml
-    dab_moving_sum_ff.xml
-    dab_ofdm_ffe_all_in_one.xml
-    dab_ofdm_sampler.xml
-    dab_ofdm_coarse_frequency_correct.xml
     dab_diff_phasor_vcc.xml
-    dab_ofdm_remove_first_symbol_vcc.xml
-#   dab_frequency_interleaver_vcc.xml
+    #   dab_frequency_interleaver_vcc.xml
     dab_qpsk_demapper_vcb.xml
     dab_complex_to_interleaved_float_vcf.xml
-    dab_modulo_ff.xml
-    dab_measure_processing_rate.xml
-    dab_fic_decode.xml
+    #   dab_fic_decode.xml
     dab_ofdm_demod.xml
     dab_ofdm_mod.xml
     dab_select_vectors.xml
-    dab_repartition_vectors.xml
-#    dab_unpuncture_vff.xml
-    dab_prune_vectors.xml
+    #    dab_unpuncture_vff.xml
     dab_fib_sink_vb.xml
-    dab_estimate_sample_rate_bf.xml
-    dab_fractional_interpolator_triggered_update_cc.xml
-    dab_magnitude_equalizer_vcc.xml
-    dab_qpsk_mapper_vbc.xml
-#   dab_ofdm_insert_pilot_vcc.xml
+    #   dab_ofdm_insert_pilot_vcc.xml
```

Comment:
completely remove such lines.

<h1>Cleanup</h1>

* especially ofdm_insert_pilot is still in the SWIG. So, it's getting exported to python, but can't be used from GRC if you're not including the XML file in the installtion. Sense?
* Your work was a rather comprehensive overhaul. Throw out stuff that's not being used anymore! I have the suspicion this applies to all the commented-out XML files here.